install_config/advanced_ldap_configuration/configuring_form_based_authentication.adoc

```diff
@@ -0,0 +1,104 @@
+= Configuring Form-Based Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon
+link:../advanced_ldap_configuration/ssd_for_ldap_failover.html[Setting
+up SSSD for LDAP Failover] and describes how to set up form-based authentication
+for signing into the {product-title} web console.
+
+[[configuring-form-based-authentication-prepare-a-login]]
+== Prepare a Login Page
+
+The {product-title} upstream repositories have a template for forms. Copy that
+to your authenticating proxy on *_proxy.example.com_*:
+
+====
+----
+# curl -o /var/www/html/login.html \
+    https://raw.githubusercontent.com/openshift/openshift-extras/master/misc/form_auth/login.html
+----
+====
```

Comment:
Tell them that they will want to modify this html file to modify the logo icon and "Welcome to _____" content for their environment.